apps/trustlab/src/components/ReportCard/ReportCard.js

```diff
@@ -31,62 +31,79 @@ const ReportCard = forwardRef(function ReportCard(props, ref) {
       sx={[{ borderRadius: "5px", p: 0 }, ...(Array.isArray(sx) ? sx : [sx])]}
       {...other}
     >
-      {image?.src && !condensed && (
-        <CardMedia
-          component="img"
-          image={image.src}
-          alt={image.alt ?? title}
-          sx={{
-            borderRadius: "10px",
-            objectFit: "cover",
-            height: 200,
-          }}
-        />
-      )}
+      <Box
+        sx={{ textDecoration: "none" }}
+        href={link?.href}
+        component={link?.href ? Link : "div"}
+      >
+        {image?.src && !condensed && (
+          <CardMedia
+            component="img"
+            image={image.src}
+            alt={image.alt ?? title}
+            sx={{
+              borderRadius: "10px",
+              objectFit: "cover",
+              height: 200,
+            }}
+          />
+        )}
+      </Box>
+
       <CardContent
         sx={{ pb: "16px !important", px: image?.src && !condensed ? 0 : 2 }}
       >
-        <Typography variant="h3" gutterBottom>
-          {title}
-        </Typography>
-        <Typography
-          variant="caption"
-          component="div"
-          sx={{
-            color: "#828499",
-            my: 2,
-          }}
+        <Box
```

Comment:
Why do we want just title and date to be clickable? Shouldn't the whole `CardContent` be clickable?... and I'm still not sure why we want to make the `CardMedia` clickability separate from `CardContent`. Shouldn't the whole `Card` or `CardActionArea` be clickable?